stock-price-checker/routes/api.js

```diff
@@ -19,7 +19,7 @@ module.exports = function (app) {
     .get(function (req, res){
       var stock = req.query.stock;
       var like = req.query.like || false;
-      var reqIP = req.connection.remoteAddress;
+      var reqIP = req.get('x-forwarded-for').split(",")[0] || req.ip.split(",")[0];
```

Comment:
Optional chaining seems to fix this while running locally:

```suggestion
      var reqIP = req.get('x-forwarded-for')?.split(",")[0] || req.ip?.split(",")[0];
```